src/status/ens.nim

```diff
@@ -172,6 +173,54 @@ proc getPrice*(): Stuint[256] =
     raise newException(RpcException, "Error getting ens username price: 0x")
   result = fromHex(Stuint[256], response.result)
 
+proc releaseEstimateGas*(username: string, address: string, success: var bool): int =
+  let
+    label = fromHex(FixedBytes[32], label(username))
+    ensUsernamesContract = contracts.getContract("ens-usernames")
+    release = Release(label: label)
+
+  var tx = transactions.buildTokenTransaction(parseAddress(address), ensUsernamesContract.address, "", "")
+  try:
+    let response = ensUsernamesContract.methods["release"].estimateGas(tx, release, success)
+    if success:
+      result = fromHex[int](response)
+  except RpcException as e:
+    raise
```

Comment:
Is it usefull to try/except without doing anything in the except block but raise, I'd assume it just re-raise the same exception catch.

Also the exception should probably be catch in the client and handle?